packages/edit-site/src/components/sidebar-navigation-data-list/style.scss

```
@@ -0,0 +1,7 @@
+.edit-site-sidebar-navigation_data-list__label {
+	color: $gray-600;
+}
+
+.edit-site-sidebar-navigation_data-list__item .edit-site-sidebar-navigation_data-list__value {
```

Tiny nit: since this uses a BEM naming convention, is the parent `.edit-site-sidebar-navigation_data-list__item` needed? Could this just be a single `.edit-site-sidebar-navigation_data-list__value` line, or does it need higher specificity?

```suggestion
.edit-site-sidebar-navigation_data-list__value {
```

---

This one is for specificity. I can change it to `.edit-site-sidebar-navigation_data-list__value.edit-site-sidebar-navigation_data-list__value` maybe to communicate that. 👍